kotlin-analysis-api/testData/annotationValue/defaultKClassValue.kt

```diff
@@ -1,8 +1,8 @@
 // TEST PROCESSOR: DefaultKClassValueProcessor
 // EXPECTED:
+// java.lang.String
 // kotlin.String
-// kotlin.String
-// kotlin.String
+// java.lang.String
```

Comment:
Given `annotation class ExampleAnnotation(val value: kotlin.reflect.KClass<*> = java.lang.String::class)`, I think the  old expected values are wrong.